src/Compilers/Core/MSBuildTaskTests/Microsoft.Build.Tasks.CodeAnalysis.UnitTests.csproj

```diff
@@ -5,7 +5,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.BuildTasks.UnitTests</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFrameworks>$(RoslynPortableTargetFrameworks)</TargetFrameworks>
```

Comment:
Hmm, thought we deleted that property. Please inline the different target frameworks here.

Reply:
We seem to still be using it in all the other unittest projects?

---
In reply to: [426953913](https://github.com/dotnet/roslyn/pull/44365#discussion_r426953913) [](ancestors = 426953913)

Reply:
Weird. I will send a PR to delete the existing usage.